### PR TITLE
Implement CSV ConfigFormatter

### LINKED ISSUE #462: Implement `CSV` ConfigFormatter
### Suggestion

introduce a `CSV` file config formatter.

### Motivation

The CSV format is widely used, and adding it as a formatter will make Configu more versatile. 

### Context

- Install the [csv](https://www.npmjs.com/package/csv) package
- Refer to this [PR](https://github.com/configu/configu/pull/253/files) as a previous formatter implementations. 

Example usage:

```bash
configu eval ... | configu export --format 'CSV'
```

### PR DESCRIPTION
**Pull request by ovindu-a**
Closes #462 

I have implemented CSV formatter as requested in the above issue.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,9 +53,12 @@
...
     "ts-node": "^10.9.2",
     "tslib": "^2.7.0",
-    "tsx": "^4.19.1",
     "tsup": "^8.3.0",
+    "tsx": "^4.19.1",
     "typescript": "^5.6.2",
     "zx": "8.1.8"
+  },
+  "dependencies": {
+    "csv-stringify": "6.5.1"
```
Comment (davidmeirlevy) [inline package.json:62]: this repository acts as a monorepo.
please install this package directly on the relevant internal package that uses it.

**Review by davidmeirlevy — CHANGES_REQUESTED** [review]
Thanks for your PR.
please follow the comment I wrote so we will be able to merge it.

Thanks! 👍

**Comment by ovindu-a** [comment]
Sure @davidmeirlevy, I will make the changes as mentioned.

File: package.json (hunk 1 above)
Comment (ovindu-a) [reply package.json:62]: @davidmeirlevy this is currently installed in the lib package. Must it be installed in a different package?

Comment (rannn505) [reply package.json:63]: @ovindu-a i guess @davidmeirlevy ment the `@configu/lib` package

**Comment by rannn505** [comment]
@ovindu-a thanks for the great effort i will merge it an fix the dependency location myself

**Comment by ovindu-a** [comment]
Sure @rannn505, let me know if there are any more issues I can work on.

**Comment by pelegpor** [comment]
Hi @ovindu-a ,

Here are the details for claiming your Hacktoberfest prizes! 🎉 Check out [#465  (comment)](https://github.com/configu/configu/issues/465#issuecomment-2498550259) for more information or simply fill out the form here: [Claim Your Swag](https://share-eu1.hsforms.com/2SVc5VKY3Thi04Bvjh17WKw2e62o0).  

Remember, you only need to fill out the form once for all your PRs. 😊

**Pull request merged**